From 46e6ce16ab6b79d9761794b51a9997a1f0f8148c Mon Sep 17 00:00:00 2001
From: Shivani Raval <sraval@habana.ai>
Date: Tue, 10 Sep 2024 13:39:47 +0300
Subject: [PATCH] [FSW-6362] build: Added changes to build mgmt io

Added zephyr makefile changes to build management for IO die.

Change-Id: Id4d9f09e9d1a30bf3e2e61cb36fa35eeee19e468
Signed-off-by: Shivani Raval <sraval@habana.ai>
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index 9b83ba0..24443f9 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -102,6 +102,10 @@
 HL_ZEPHYR_PACKAGES += mgmt
 endif
 
+ifdef BR2_PACKAGE_MGMT_IO
+HL_ZEPHYR_PACKAGES += mgmt
+endif
+
 ifdef BR2_PACKAGE_MGMT_COMPUTE
 HL_ZEPHYR_PACKAGES += mgmt_compute
 endif
